shoop/admin/modules/orders/dashboard.py

```diff
@@ -47,7 +47,7 @@ def get_chart(self):
         bar_chart.add_data(
             _("Sales (%(currency)s)") % {"currency": self.currency},
             [
-                bankers_round(v["sum"], settings.SHOOP_ORDER_TOTAL_DECIMALS)
+                bankers_round(v["sum"], 2)
```

Comment:
This probably oughta have some sort of `# TODO: To be fixed in SHOOP-1912` thing, then?